access/types/forms.py

```diff
@@ -499,27 +510,40 @@ def row_options(self, configuration, row):
     def grade_checkbox(self, configuration, value, hints=None):
         hints = hints or []
         correct_count = 0
-
-        # All correct required if any configured
+        wrong_answers = 0
+        non_neutral_count = 0
+        # All correct required if any configured and partial-points not set
```

Comment:
`# All correct required if any configured and partial-points not set`
Is the comment correct anymore? Does the new implementation fail all submissions if the config has no correct choices set (in normal non-feedback questionnaires)? I think we decided to make it so that no submissions are accepted if the teacher forgets to mark up the correct choices. Previously, the grader awarded any submission full points in that case.

Reply:
Now it will award empty checkbox with full points. If there are correct choices, every one of the must be chosen to get any points, if partial-points is not set

Reply:
Right, if no correct choices are set, then the student gets points by submitting an empty solution. At any rate, the comment in the code is hard to understand and it should be written more clearly.